doc/getting_started/_index.md

```diff
@@ -0,0 +1,198 @@
+
+*Skip this step if using the Docker container.*
+
+To build device software you need a baremetal RISC-V toolchain.
+Even if you already have one installed, we recommend using the prebuilt toolchain provided by lowRISC, because it is built with the specific patches and options that OpenTitan needs.
+You can install the toolchain using the `util/get-toolchain.py` script, which will download and install the toolchain to the default path, `/tools/riscv`.
+
+```console
+$ cd $REPO_TOP
+$ ./util/get-toolchain.py
+```
+
+If you got no errors from that script, **you're done and can go to step 4**. If there were problems, read on.
+
+#### Troubleshooting
+
+If you need to install to a different path than `/tools/riscv` (for instance, if you do not have permission to write to the `/tools` directory), then you can specify a different location using the `--install-dir` option.
+Run `./util/get-toolchain.py --help` for details.
+You can alternatively download the file starting with `lowrisc-toolchain-rv32imcb-` from [GitHub releases](https://github.com/lowRISC/lowrisc-toolchains/releases/latest) and unpack it to the desired installation directory.
+
+Assuming one of the above worked and you have installed to a non-standard location, you will need to set the `TOOLCHAIN_PATH` environment variable to match whatever path you used.
+For example, if I wanted to install to `~/ot_tools/riscv`, then I would use:
+```console
+$ ./util/get-toolchain.py --install_dir=~/ot_tools/riscv
+$ export TOOLCHAIN_PATH=~/ot_tools/riscv
+```
+Add the `export` command to your `~/.bashrc` or equivalent to ensure that the `TOOLCHAIN_PATH` variable is set for future sessions.
+Check that it worked by opening a new terminal and running:
+```console
+$ ls $TOOLCHAIN_PATH/bin/riscv32-unknown-elf-as
+```
+If that prints out the file path without errors, then you've successfully installed the toolchain.
+Otherwise, try to find the `riscv32-unknown-elf-as` file in your file system and make sure `$TOOLCHAIN_PATH` is correctly set.
+
+## Step 4: Build OpenTitan Software
+
+Follow the [dedicated guide]({{< relref "build_sw" >}}) to build OpenTitan's software, and then return to this page.
+
+## Step 5: Set up your Simulation Tool or FPGA
+
+*Note: If you are using the pre-built Docker container, Verilator is already installed.
+Unless you know you need the FPGA or DV guides, you can skip this step.*
+
+In order to run the software we built in the last step, we need to have some way to mimic an OpenTitan chip.
+There are a few different options depending on your equipment and use-case.
+Follow the guide that applies to you:
+* **Option 1 (default, recommended for new users):** [Verilator guide]({{< relref "setup_verilator.md" >}}), or
+* Option 2 (FPGA setup): [FPGA guide]({{< relref "setup_fpga.md" >}}), or
+* Option 3 (design verification setup): [DV guide]({{< relref "setup_dv.md" >}})
+* Option 4 (formal verification setup): [Formal Verification guide]({{< relref "setup_formal.md" >}})
```

Comment:
Optional steps makes sense, yep! For the other optional steps, I have a little blurb saying "if X, then follow step Y to do Z". For the formal verification flow, what should X be? "If you want to set up formal verification" is the best I can come up with but it would be better to have something a little more informative :slightly_smiling_face: